src/main/java/ro/secur/auth/configuration/SecurityConfiguration.java

```diff
@@ -52,6 +60,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers(RESET_PASSWORD_URL, FORGOT_PASSWORD_URL, USERS_URL).permitAll()
                 .and()
+                .exceptionHandling().authenticationEntryPoint(jwtAuthenticationEntryPoint)
```

Comment:
Is there a reason behind having duplicate lines : line 56 and this one seems to be the same..

Reply:
trebuie sa imi iau ochelari, ma uitam dupa linia aia si nu o vedeam